.travis.yml

```diff
@@ -18,11 +18,8 @@ cache:
     - $HOME/.npm
 
 before_install:
-  - pecl install xdebug-$XDEBUG_VERSION
-  - phpenv config-add travis-php.ini
-  - nvm install $NODE_VERSION
-  - nvm use $NODE_VERSION
-  - npm install -g npm@$NPM_VERSION --depth 0
+  - chmod ugo+x ./scripts/travis-beforeinstall.sh
```

Comment:
Why not just `chmod` the checked-in file?

Reply:
I'll change it. But I'd rather just throw out macos support and then there's almost no need for this script.. As the Go Proverb goes, "A little copying is better than a little dependency"...

Reply:
Did trow out macos and with it this huge script. The few lines that linux needs for initialization are back in the yml.